app/src/main/java/com/astasko/pdp/app/user/controller/UserController.java

```diff
@@ -0,0 +1,34 @@
+package com.astasko.pdp.app.user.controller;
+
+import com.astasko.pdp.domain.model.User;
+import com.astasko.pdp.service.api.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+
+@Controller
+public class UserController {
+
+    @Autowired
+    private UserService userService;
+
+    @GetMapping("/user/{id}")
```

Comment:
The value of @GetMapping can be simplified to "/{id}" after you will add @RequestMapping("/user") to the class.